tuner/examples/test/tuner_test.py

```diff
@@ -33,6 +106,53 @@ def main():
     if stop_after_phase == libtuner.ExecutionPhases.generate_candidates:
         return
 
+    test_tuner = TestTuner()
+    print("Compiling candidates...")
+    compiled_candidates = libtuner.compile(
+        args, path_config, candidates, candidate_trackers, test_tuner
+    )
+
+    print("Benchmarking compiled candidates...")
+    top_candidates = libtuner.benchmark(
+        args,
+        path_config,
+        compiled_candidates,
+        candidate_trackers,
+        test_tuner,
+        args.test_num_dispatch_candidates,
+    )
+
+    print("Compiling models with top candidates...")
+    test_tuner.compile_flags = [
+        "--iree-hal-target-backends=rocm",
+        "--iree-hip-target=gfx942",
+    ]
```

Comment:
Since we know how to compile the model, could we generate the benchmark files for the user to skip manual steps? Doesn't have to be in this PR, but it would be nice to have a TODO.

Reply:
Yeah, that's a nice idea, and I think it's better as a followup.